modules/rest-api/src/main/java/org/wso2/carbon/mdm/api/Operation.java

```diff
@@ -134,4 +134,47 @@ public Message installApplication(ApplicationWrapper applicationWrapper) throws
             throw new MDMAPIException(msg, e);
         }
     }
+
+	@POST
+	@Path("uninstallApp/{tenantDomain}")
+	public Message uninstallApplication(ApplicationWrapper applicationWrapper,
+										@PathParam("tenantDomain") String tenantDomain) throws MDMAPIException {
+
+		Message responseMsg = new Message();
+		AppManagerConnector appManagerConnector;
+		DeviceManagementService deviceManagementService;
+		org.wso2.carbon.device.mgt.common.operation.mgt.Operation operation = null;
+		ArrayList<DeviceIdentifier> deviceIdentifiers;
+		try{
+			appManagerConnector = MDMAPIUtils.getAppManagementService(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME);
+			deviceManagementService =  MDMAPIUtils.getDeviceManagementService(MultitenantConstants
+					.SUPER_TENANT_DOMAIN_NAME);
+			MobileApp mobileApp = applicationWrapper.getApplication();
+
+			if (applicationWrapper.getDeviceIdentifiers() != null){
+				for(DeviceIdentifier deviceIdentifier : applicationWrapper.getDeviceIdentifiers()){
```

Comment:
Fixed in Commit [https://github.com/wso2/product-mdm/commit/1a4723b3b6cb85b70d5cfd88ebd7f37ea16e564f].